setup.py

```diff
@@ -7,14 +7,14 @@
 
 extras_require = {
     "test": [
-        "pytest>=6.2.5",
-        "pytest-xdist>=2.4.0,<3",
-        "tox==3.14.6",
+        "pytest>=7.0.0",
+        "pytest-xdist>=2.4.0",
     ],
     "lint": [
-        "flake8==3.7.9",
+        "flake8>=5.0.0",
```

Comment:
Flake8 [says](https://flake8.pycqa.org/en/latest/internal/releases.html) they follow semver. At the very least we've had it pinned for a long time, let's try a move to just a lower bound.

Reply:
The thing you run into is that a new minor release can make code start failing flake8. It's not that the flake8 API changed exactly, they just start warning about something new (I think this is what they're getting at with ["new errors detected by dependencies"](https://flake8.pycqa.org/en/latest/internal/releases.html#minor-releases)). So a new flake8 can cause all your repositories to simultaneously start failing every PR pull.

That might mean that a PR that's green will show as a red failure on master if flake8 is released between the last PR push and the merge commit. If there's some reason that you need to push a PR quickly, and a failure happens, you have to hunt down what happened.

The upside of getting the very latest flake8 detections does not seem worth that sporadic pain, IMO.

Reply:
I see the point for sure though I do wonder how often that would happen. I think, since we have a strong voice in favor of one way, I end up leaning towards a re-pin here in the template repo. The version could probably use an update though!

Reply:
Got it. Will pin `flake8` and `flake8-bugbear` going forward.